bin/cli-serve

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env node
+
+/**
+ * Setup the debug paramater.
+ */
+
+process.env.DEBUG = process.env.TALK_DEBUG;
+
+const app = require('../app');
+const debug = require('debug')('talk:server');
+const http = require('http');
+const init = require('../init');
+const port = normalizePort(process.env.TALK_PORT || '3000');
+const scraper = require('../services/scraper');
+const mongoose = require('../mongoose');
+const util = require('../util');
+
+/**
+* Get port from environment and store in Express.
+*/
+app.set('port', port);
+
+/**
+* Create HTTP server.
+*/
+const server = http.createServer(app);
+
+/**
+ * Event listener for HTTP server "error" event.
+ */
+function onError(error) {
+  if (error.syscall !== 'listen') {
+    throw error;
+  }
+
+  let bind = typeof port === 'string'
+    ? `Pipe ${port}`
+    : `Port ${port}`;
+
+  // handle specific listen errors with friendly messages
+  switch (error.code) {
+  case 'EACCES':
+    console.error(`${bind} requires elevated privileges`);
+    break;
+  case 'EADDRINUSE':
+    console.error(`${bind} is already in use`);
+    break;
+  }
+
+  throw error;
```

Comment:
Well, we want it to throw the error either way, that way the application will close.